content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -252,6 +277,12 @@ Docker Engine with Kubernetes.
 
 For `cri-dockerd`, the CRI socket is `/run/cri-dockerd.sock` by default.
 
+#### Overriding the sandbox (pause) image
+
+To overwrite the sandbox (pause) image the `cri-dockerd` adapter
+[accepts a flag](https://github.com/Mirantis/cri-dockerd/blob/master/src/config/options.go)
+to configure this `--pod-infra-container-image`.
```

Comment:
```suggestion
To overwrite the sandbox (pause) image the [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter
accepts the `--pod-infra-container-image` flag.
```

instead of linking to the source code, suggesting that we link to the main `cri-dockerd` page with hopes that its documentation will enumerate flags eventually.